commands-v2/ramsete/robotcontainer.py

```diff
@@ -0,0 +1,92 @@
+from commands2 import RunCommand, RamseteCommand
+from commands2.button import JoystickButton
+
+from wpilib import XboxController
+from wpilib.controller import RamseteController, PIDController
+
+from wpimath.controller import SimpleMotorFeedforwardMeters
+
+from wpilib.kinematics import ChassisSpeeds
+
+from wpilib.interfaces import GenericHID
+
+from wpimath.trajectory.constraint import DifferentialDriveVoltageConstraint
+from wpimath.trajectory import TrajectoryConfig, TrajectoryGenerator, Trajectory
+
+from wpimath.geometry import Pose2d, Rotation2d, Translation2d
+
+from subsystems.drivetrain import Drivetrain
+
+from constants import constant
+
+
+class RobotContainer:
+    def __init__(self):
+
+        self.driverController = XboxController(constant.driverControllerID)
+        self.driveSubsystem = Drivetrain(self.driverController)
+
+        self.configureButtons()
+
+        doThis = RunCommand(self.driveSubsystem.arcadeDrive, self.driveSubsystem)
+
+        self.driveSubsystem.setDefaultCommand(doThis)
+
+    def getAutonomousCommand(self):
+        motorFFOne = SimpleMotorFeedforwardMeters(
+            constant.ksVolts,
+            constant.kvVoltSecondsPerMeter,
+            constant.kaVoltSecondsSquaredPerMeter,
+        )
+
+        autoVoltageConstraint = DifferentialDriveVoltageConstraint(
+            motorFFOne, constant.driveKinematics, 10  # 10 volts max.
+        )
+
+        config = TrajectoryConfig(
+            constant.maxSpeedMetersPerSecond,
+            constant.maxAccelerationMetersPerSecondSquared,
+        )
+
+        config.setKinematics(constant.driveKinematics)
+        config.addConstraint(autoVoltageConstraint)
+
+        initial = Pose2d(0, 0, Rotation2d(0))
+        movements = [Translation2d(50, 50)]
```

Comment:
These values are different than what Java uses?

Reply:
Yeah, I was testing the code and I suppose I forgot to change it. I exaggerated it so it was easier for me to see. I'll change that too